src/utils/clmfates_interfaceMod.F90

```diff
@@ -863,8 +863,13 @@ subroutine dynamics_driv(this, nc, bounds_clump,      &
          nlevsoil = this%fates(nc)%bc_in(s)%nlevsoil
 
          ! Decomposition fluxes
-         this%fates(nc)%bc_in(s)%w_scalar_sisl(1:nlevsoil) = soilbiogeochem_carbonflux_inst%w_scalar_col(c,1:nlevsoil)
-         this%fates(nc)%bc_in(s)%t_scalar_sisl(1:nlevsoil) = soilbiogeochem_carbonflux_inst%t_scalar_col(c,1:nlevsoil)
+         if ( decomp_method /= no_soil_decomp )then
+            this%fates(nc)%bc_in(s)%w_scalar_sisl(1:nlevsoil) = soilbiogeochem_carbonflux_inst%w_scalar_col(c,1:nlevsoil)
+            this%fates(nc)%bc_in(s)%t_scalar_sisl(1:nlevsoil) = soilbiogeochem_carbonflux_inst%t_scalar_col(c,1:nlevsoil)
+         else
+            this%fates(nc)%bc_in(s)%w_scalar_sisl(1:nlevsoil) = 0.0_r8
+            this%fates(nc)%bc_in(s)%t_scalar_sisl(1:nlevsoil) = 0.0_r8
+         end if
```

Comment:
@rgknox this part here got me wondering if we are going to pass zero moisture and temp scalars (which gets used to calculate the fates fragmentation scalar, should we be doing a check against the `hlm_decomp` and `hlm_use_sp` on the fates side to avoid the fragmentation scalar (and other related routines)?  Or add some new namelist switch to allow for soilbgc to be run with fates-sp or not?